packages/gestalt/src/PageHeader.js

```diff
@@ -0,0 +1,72 @@
+// @flow strict
+import React, { type Node, Children } from 'react';
+import Box from './Box.js';
+import Flex from './Flex.js';
+import Heading from './Heading.js';
+import Text from './Text.js';
+import { type Dimension } from './boxTypes.js';
+
+type Props = {|
+  title: string,
+  maxWidth?: Dimension,
+  primaryAction?: Node,
+  secondaryAction?: Node,
+  subtext?: string,
+|};
+
+export default function PageHeader({
+  maxWidth = '100%',
+  primaryAction,
+  secondaryAction,
+  subtext,
+  title,
+}: Props): Node {
+  const renderAction = (action?: Node): Node => {
```

Comment:
We typically want to avoid sub-render methods like this. If something is separate enough to necessitate a method, it should be a separate component.